src/main/java/teammates/storage/search/SearchManager.java

```diff
@@ -69,21 +72,32 @@ SolrQuery getBasicQuery(String queryString) {
         return query;
     }
 
-    QueryResponse performQuery(SolrQuery query) throws SearchNotImplementedException {
+    QueryResponse performQuery(SolrQuery query) throws SearchServiceException {
         if (client == null) {
-            throw new SearchNotImplementedException();
+            throw new SearchServiceException("Search service is not implemented.", HttpStatus.SC_NOT_IMPLEMENTED);
```

Comment:
Even though it's written like this before, let's rephrase this a bit to "~Search service~Full-text search is not ~implemented~ available".